Roles/Impostor/RiftMaker.cs

```diff
@@ -113,7 +113,8 @@ public static void ReceiveRPC(MessageReader reader)
     public static void ApplyGameOptions()
     {
         AURoleOptions.ShapeshifterCooldown = SSCooldown.GetFloat();
-        AURoleOptions.ShapeshifterLeaveSkin = true;
+        //AURoleOptions.ShapeshifterLeaveSkin = true;
+        //You set leave skin here only influences the rifter it own. Its not global
         AURoleOptions.ShapeshifterDuration = 1f;
```

Comment:
What? It should be because for Rift Maker its helps a lot to determine exactly where he placed the marker.

Reply:
huh, but if we reject the shapeshift animation, there wont be skin left

Reply:
Bad